binary-search.json

```diff
@@ -0,0 +1,60 @@
+{
+    "#": [
+        "Here -1 is used to indicate that the value is not included in the array.",
+        "It should be replaced with the respective expression that is idiomatic",
+        "for the language that implements the tests.",
+
+        "Following https://github.com/exercism/x-common/issues/234 the exercise",
+        "should NOT include checking whether the array is sorted as it defeats",
+        "the point of the binary search.",
+
+        "The exercise should utilize an array-like (i.e. constant-time indexed)",
+        "data structure and not a linked list. If something like an array is not",
+        "usually used in the language the problem should not be implemented.",
+        "See https://github.com/exercism/x-common/issues/244 for details."
+    ],
+    "cases": [
+        {
+            "description": "finds a value in an array with one element",
+            "array": [6],
+            "value": 6,
+            "expected": 0
+        },
+        {
+            "description": "finds a value in the middle of an array",
+            "array": [1, 3, 4, 6, 8, 9, 11],
+            "value": 6,
+            "expected": 3
+        },
+        {
+            "description": "finds a value at the beginning of an array",
+            "array": [1, 3, 4, 6, 8, 9, 11],
+            "value": 1,
+            "expected": 0
+        },
+        {
+            "description": "finds a value at the end of an array",
+            "array": [1, 3, 4, 6, 8, 9, 11],
+            "value": 11,
+            "expected": 6
+        },
+        {
+            "description": "finds a value in the middle of an array of odd length",
+            "array": [1, 3, 5, 8, 13, 34, 34, 55, 89, 144, 233, 377, 634],
```

Comment:
Also you probably need test cases for when the 'correct' duplicated element is the first element as well so you can't fake it by reversing the list or taking the highest index or some other similar workaround.